packages/app/src/app/store/modules/server/actions.js

```diff
@@ -0,0 +1,20 @@
+import { dispatch } from 'codesandbox-api';
+
+export function restartSandbox({ executor }) {
+  executor.emit('sandbox:restart');
+}
+
+export function restartContainer({ executor }) {
+  executor.emit('sandbox:restart-container');
+}
+
+/**
+ * We need to remove this logic later! We need to move it to the terminal component
+ * and let that component listen to the executor directly.
+ */
+export function logSandboxMessage({ props }) {
```

Comment:
@CompuIves This can be an arrow function

```suggestion
export const logSandboxMessage = ({ props }) => {
```

Reply:
This is mostly semantics, I don't think it's that important to use a const here.

Reply:
Just because I see arrow functions are more commonly used in the community